suite-native/module-settings/src/components/ApplicationSettings.tsx

```diff
@@ -9,12 +10,14 @@ import {
     RootStackParamList,
     RootStackRoutes,
 } from '@suite-native/navigation';
-import { isDevelopOrDebugEnv } from '@suite-native/config';
 
 import { SettingsSection } from './SettingsSection';
 import { SettingsSectionItem } from './SettingsSectionItem';
+import { isDevButtonVisibleAtom } from './ProductionDebug';
 
 export const ApplicationSettings = () => {
+    const [isDevButtonVisible] = useAtom(isDevButtonVisibleAtom);
```

Comment:
you can use `useAtomValue` if you don't need change it's value

Reply:
Forgot about this feature. Implemented